CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributions Welcome
+
+First off, thank you for considering contributing to the Github-Webhook-Resource! This resource is a simple project, but has plenty of room for improvement. Many features of Github's webhook implementation are yet to be built into the Github-Webhook-Resource, and all contributions are welcome.
+
+If you're just looking for quick feedback for an idea or proposal, feel free to open an
+[issue](https://github.com/homedepot/github-webhook-resource/issues).
+
+Follow the [contribution workflow](#contribution-workflow) for submitting your
+changes to the Github-Webhook-Resource codebase. If you want to receive high-level but still
+commit-based feedback for a contribution, follow the
+[request for comments](#request-for-comments) steps instead.
+
+## Contribution Workflow
+
+The Github-Webhook-Resource uses the “fork-and-pull” development model. Follow these steps if
+you want to merge your changes to the Github-Webhook-Resource:
+
+1. Within your fork of the
+   [Github-Webhook-Resource](https://github.com/homedepot/github-webhook-resource), create a
+   branch for your contribution. Use a meaningful name.
+2. Create your contribution, meeting all
+   [contribution quality standards](#contribution-quality-standards)
+3. [Create a pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/)
+   against the master branch of the Github-Webhook-Resource repository.
+4. Add a reviewer to your pull request. Work with your reviewer to address any comments and obtain an approval.
+   To update your pull request amend existing commits whenever applicable and
+   then push the new changes to your pull request branch.
+5. Once the pull request is approved, one of the [maintainers](MAINTAINERS.md). will merge it.
+
+## Request for Comments
```

Comment:
For this particular repo, I suggest we remove the entire **Request for Comments** section

Reply:
I think I understand why this is a good idea, but could you share your thoughts behind this change?